src/index.js

```diff
@@ -12,6 +12,7 @@ import WithStylesContext from './WithStylesContext';
 import {getTags, getLessons} from './util';
 
 const iconContext = require.context('lessonSrc/', true, /^\.\/[^\/]*\/logo-black\.png/);
+const playlistContext = require.context('onlyContent!lessonSrc//', true, /^\.\/[^\/]*\/playlists\/[^\/]*\.txt$/);
```

Comment:
Må dette parses gjennom markdown-it? Så vidt jeg kan se, er det bare linjer med paths til andre markdown-filer? Da kan kanskje filen requires inn som raw-text e.l. istedet (bytt ut onlyContent med raw ---  https://github.com/webpack/raw-loader)

Forøvrig, det skal vel bare være én slash etter lessonSrc?